### ISSUE TITLE
Add or update CADD-splice in silico annotations 

### ISSUE THREAD

**Issue by anneodonnell** [broadinstitute/hail-elasticsearch-pipelines#317]
I can't tell if this is adding CADD splice or just making sure we are using the most recent version of CADD annotations. 
The readme implies it is integrated into the general CADD scores with v1.5 (though v1.6 may be the current version)

CADD-Splice—improving genome-wide variant effect prediction using deep learning-derived splice scores (2021)
[PMID: 33618777](https://pubmed.ncbi.nlm.nih.gov/33618777/)
Specific to splice site variants and intronic variants. 
Integrates SpliceAI and MMSplice 
Pre-computed files: https://cadd.gs.washington.edu/download
Script for offline scoring: https://github.com/kircherlab/CADD-scripts/

**Comment by hanars** [comment]
Our general approach with new/ updated scores is that they get rolled out as we naturally reload data for projects with new data, we don't go out and proactively try to reload everything. I think that should be sufficient here - we can always revisit if after we have added a lot of new scores it would be worth a one-time update, but that would be a lot of work and money so that would need to be very infrequent

**Comment by anneodonnell** [comment]
Agreed - I just wanted to ensure we were using the updated CADD as it incorporates splicing. So this sounds good - thanks!

**Comment by mike-w-wilson** [comment]
I confirmed that CADD-Splice was the model used for CADD v1.6 which is the version we are currently using. CADD-Splice incorporates SpliceAI and MMSplice as features in the v1.6 model. All projects re/loaded since January will automatically be using CADD-Splice.
